StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostController.swift

```diff
@@ -70,7 +70,10 @@ extension HostController: HostViewControllerDelegate {
         _ viewController: HostViewController,
         didFetch synchronizePayload: FinancialConnectionsSynchronize
     ) {
-        guard let consentPaneModel = synchronizePayload.text?.consentPane else {
+        guard
+            let consentPaneModel = synchronizePayload.text?.consentPane,
+            let visualUpdate = synchronizePayload.visual
```

Comment:
Did this to avoid propagating unnecessary nulls in Native code. This may change on backend to be non-null:
- https://stripe.slack.com/archives/C02KW8G938W/p1671732339796429